Docs/Getting Started with the GE Reflective Memory Component.md

```diff
@@ -10,61 +10,59 @@ Before starting this exercise, install the following software:
-   - Enter "GE Reflective Memory" in the first element of the components array
+1. Select the newly created Data Sharing Framework custom device
+1. Click **New** to launch a dialog box to manually configure the custom device by setting the values in the DSF configuration cluster.
+   - Enter *GE Reflective Memory* in the first element of the components array
       - **Thread(0).transfer groups(0)**
-         - Verify **core.direction** = "Tx"
+         - Verify **core.direction** = *Tx*
          - Note that the channels array is configured to include a single numeric element
-            - **engine data type** = "DBL" and should not be changed.
-            - **string data type**= "DBL" and may be set to any datatype wide enough to hold the data value to transfer
+            - **engine data type** = *DBL* and should not be changed.
+            - **string data type**= *DBL* and may be set to any datatype wide enough to hold the data value to transfer
             ![](support/GERMConfigureTxTransfer.png)
 
-       - **Thread(0).transfer groups(1)** _(Navigate to index 1 using the horizontal scrollbar for the **threads** array)_
-          - Verify **core.direction** = "Rx"
+       - **Thread(0).transfer groups(1)** _(Navigate to index 1 using the horizontal scrollbar for the threads array)_
+          - Verify **core.direction** = *Rx*
          - Note that the channels array is configured to include a single numeric element
-            - **engine data type** = "DBL" and should not be changed.
-            - **string data type** = "DBL" and should be configured exactly the same as the 'Tx' transfer.
+            - **engine data type** = *DBL* and should not be changed.
+            - **string data type** = *DBL* and should be configured exactly the same as the Tx transfer.
             ![](support/GERMConfigureRxTransfer.png)
-            
-    - Press **Apply** to save your new configuration
-    - Close the dialog
-1. In the configuration tree, under "[0] Plugin", select "[0] Thread". In the configuration page:
-   - Select "GE Reflective Memory" in **Components** listbox
-   - Set **VISA Resource** to the **VISA Resource Name** as displayed in NI-MAX when the GE Reflective Memory Card is selected
-   - No other configuration needs to be modified for this exercise.
-1. Save the System Definition
-1. Close the System Explorer
-1. Add a new 'Screen'
-1. Expand the System Definition tree and add the VeriStand channels for the configured transfers
+
+    - Click **Apply** to save your new configuration
+    - Close the dialog box
+1. In the configuration tree, under [0] Plugin, select **[0] Thread** to open the configuration page:
+   - In Components listbox, select **GE Reflective Memory**
+   - Set VISA Resource to the VISA Resource Name
+   **Note** You can find the Visa Resource Name in NI-MAX by selecting the GE Reflective Memory Card
```

Comment:
Another mid-sentence "note"

Reply:
I'll add a period to the end of that sentence so the note does not look mid-sentence.